src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/Location.js

```diff
@@ -48,6 +57,32 @@ export const Location = ({
     `${TRANSLATION_ID_PREFIX}.order.location.field.quantityElectronic.info`,
   );
 
+  const handleLocationAdd = useCallback(
+    () => {
+      const onLocationAdd = (fieldsPath, refTable, fieldIndex, isFirstSubfield) => {
+        const repeatableFieldActionPath = getRepeatableFieldName(fieldIndex);
+
+        handleRepeatableFieldAndActionAdd(repeatableFieldActionPath, fieldsPath, refTable, setReferenceTables, isFirstSubfield);
+      };
+
+      return onAdd(locations, 'locations', LOCATION_FIELDS_MAP.LOCATIONS, initialFields, onLocationAdd, 'order');
+    },
+    [initialFields, locations, setReferenceTables],
```

Comment:
Add `LOCATION_FIELDS_MAP.LOCATIONS` to the dependency array here and elsewhere where lint whines about new code.